### PR TITLE
Update Microsoft Azure OAuth guide

### PR DESCRIPTION
**Pull request by S3Prototype**
This PR:

- Removes screenshots
- Updates instructions for new Azure UI
- Updates description of Active Directory in our docs. Microsoft [rebranded this feature](https://learn.microsoft.com/en-us/entra/fundamentals/new-name) to Entra ID

🔎 Preview: https://clerk.com/docs/pr/1135/authentication/social-connections/microsoft

### CODE REVIEW

File: docs/authentication/saml/azure.mdx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
-title: Use Microsoft Azure AD for SAML SSO
-description: Learn how to integrate Microsoft Azure AD with Clerk using SAML SSO.
+title: Use Microsoft Entra ID for SAML SSO
+description: Learn how to integrate Microsoft Azure Entra ID with Clerk using SAML SSO.
```
Comment (S3Prototype) [inline docs/authentication/saml/azure.mdx:3]:
```suggestion
description: Learn how to allow users to sign into your Clerk app with their Microsoft account using SAML SSO.
```

File: docs/authentication/social-connections/microsoft.mdx [hunk 2]
```diff
@@ -1,115 +1,128 @@
 ---
-title: Microsoft
 description: Learn how to set up social connection with Microsoft.
```
Comment (S3Prototype) [inline docs/authentication/social-connections/microsoft.mdx:2]:
```suggestion
description: Learn how to allow users to sign into your Clerk app with their Microsoft account using OAuth.
```

File: docs/authentication/social-connections/microsoft.mdx [hunk 3]
```diff
@@ -1,115 +1,128 @@
...
+
+</TutorialHero>
+
+Enabling OAuth with Microsoft Azure Entra ID (formerly [Active Directory](https://learn.microsoft.com/en-us/entra/fundamentals/new-name)) allows your users to sign in and sign up to your Clerk application with their Microsoft account.
 
 Adding social connection with Microsoft Azure to your app with Clerk only requires populating the **Client ID**, **Client Secret** and **Authorized redirect URI** in your instance settings.
 
 To make the development flow as smooth as possible, Clerk uses preconfigured shared OAuth credentials and redirect URIs for development instances - no other configuration is needed.
 
 For production instances, you will need to generate your own Client ID and Client Secret using your Microsoft Azure account.
```
Comment (S3Prototype) [inline docs/authentication/social-connections/microsoft.mdx:33]:
```suggestion
Adding social connection with Microsoft Azure to your app with Clerk only requires populating the **Client ID**, **Client Secret** and **Authorized redirect URI** in your instance settings.

Clerk uses preconfigured shared OAuth credentials and redirect URIs for development instances - no other configuration is needed. For production instances, you must generate your own **Client ID** and **Client Secret** as shown in the instructions in this guide.

```

File: docs/authentication/social-connections/microsoft.mdx [hunk 4]
```diff
@@ -1,115 +1,127 @@
...
-In the new application form, make sure you fill in the following fields:
-
-![Filling required fields](/docs/images/authentication-providers/microsoft/1a5dc8db362fc73ca1791d13fc6fb2b74f9f2b81-3360x1782.png)
-
-After creating a new app, you will be presented with the app information page.
-
-The **Application (client) ID** visible on this page should be pasted into the **Client ID** form field on the Clerk end.
-
-To generate a value, click on the **Client credentials** link:
+Adding social connection with Microsoft Azure to your app with Clerk only requires populating the **Client ID**, **Client Secret** and **Authorized redirect URI** in your instance settings.
```
Comment (S3Prototype) [inline docs/authentication/social-connections/microsoft.mdx:29]:
```suggestion
Configuring this feature requires populating the **Client ID**, **Client Secret** and **Authorized redirect URI** in your Clerk app's settings.

```

**Review by alexisintech — CHANGES_REQUESTED** [review]

File: docs/authentication/social-connections/microsoft.mdx [hunk 5]
```diff
@@ -1,115 +1,128 @@
...
-At the time of writing, Clerk supports only Client secrets.
+1. From your app's overview, copy the **Application (client) ID** and save it somewhere secure. It's required connecting your Microsoft Entra ID app to your Clerk application.
+1. On this same page, under **Client credentials**, select **Add a certificate or secret** to generate a Client Secret.
+1. You'll be taken to the **Certificate & secrets** tab, where you can select the **New client secret** button. In the modal that appears, add a description and set an expiration time for your secret.
+    > [!IMPORTANT]
+    > When your secret expires, your social connection will stop working until you generate a new client secret and add it to your Clerk app.
+1. Copy the test in your new client secret's **Value** field and save it somewhere secure. You'll add it to your Clerk application later, alongside your client ID.
 
-To create a new client secret, click on the New client secret button as illustrated below:
+### Connect Entra ID app and get your redirect URI
```
Comment (alexisintech) [inline docs/authentication/social-connections/microsoft.mdx:63]:
```suggestion
### Connect your Entra ID app and get your redirect URI

```

File: docs/authentication/social-connections/microsoft.mdx [hunk 6]
```diff
@@ -1,115 +1,128 @@
...
+    - Under **Client ID**, add the value you copied from **Application (client) ID** in the Entra ID dashboard.
+    - Under **Client Secret**, add the client secret value you generated in the Entra ID dashboard.
+    - Copy the **Authorized redirect URI**. You need it for the final step to configure your Entra ID app.
+    - Select **Save**.
 
-You will be prompted to provide a name and lifetime for the new secret.
+### Enable OpenID
 
-![Adding a name and selecting expiration date](/docs/images/authentication-providers/microsoft/08f6445903a233c89ad96be25de3293def891cb6-1172x854.png)
+For OAuth to function, Clerk requires an `id_token` from Microsoft Entra ID.
```
Comment (alexisintech) [inline docs/authentication/social-connections/microsoft.mdx:75]: We mention `id_token` here but the user doesn't use this, do they? I'm not sure why we mention it.

We can just say something like "To connect your Clerk app to your Microsoft app, you must set the **Authorized redirect URI** in your Microsoft dashboard.

File: docs/authentication/social-connections/microsoft.mdx [hunk 7]
```diff
@@ -1,115 +1,128 @@
...
-After confirming your choice, the new secret will be listed. This is the only time you will able to copy & paste it, so make sure you do so before navigating away. Also, take note of the lifetime of said secret because you will need to generate a new one before it expires.
+1. Return to the tab where your Microsoft Entra ID application dashboard is open.
+1. In the sidebar, select **Authentication**, then select **Add a platform**.
+1. In the **Redirect URIs** field of the modal that appears, add your Clerk application's authorized redirect URI. Do the same in the **Front-channel logout URL** field.
 
-The newly generated secret can now be pasted in the **Client Secret** input on the Clerk end.
+### Test your OAuth
 
-Finally, you'll need to instruct your Microsoft Azure app to operate in OpenID  mode so that it provides clerk with an **id_token** when the sign-in completes. This allows Clerk to populate data about the user from their Microsoft profile.
+The simplest way to test your OAuth is to visit your Clerk application's account portal, which Clerk creates for all applications.
```
Comment (alexisintech) [inline docs/authentication/social-connections/microsoft.mdx:83]: Two suggestions here based on where we want to say "out-of-the-box"

```suggestion
The simplest way to test your OAuth is to visit your Clerk application's [Account Portal](/docs/account-portal/overview), which is available for all Clerk applications out-of-the-box.

```

```suggestion
The simplest way to test your OAuth is to visit your Clerk application's [Account Portal](/docs/account-portal/overview), which is available out-of-the-box for all Clerk applications.

```

**Review by alexisintech — APPROVED** [review]

**Pull request merged**